macros/edr/materializations/model/metrics.sql

```diff
@@ -22,6 +22,30 @@
   {% do return(metrics) %}
 {% endmacro %}
 
+{% macro bigquery__query_table_metrics() %}
+    select
+      {{ modules.datetime.datetime.utcnow().timestamp() }} as build_timestamp,
+      row_count
+    from {{ this.schema }}.__TABLES__
+    where table_id = '{{ this.name }}'
+  {% endset %}
+
+  {% set metrics = [] %}
+  {% for metric_column in elementary.run_query(query).columns %}
+    {% set metric_name = metric_column.name %}
+    {% set metric_value = metric_column[0] %}
```

Comment:
Can you add a safety that the query actually returns any results? (it should, but mainly to avoid edge cases / never fail here)

Reply:
can you clarify what is the expected behavior? if `elementary.run_query(query).columns` returns 0 columns, then it just wouldn't update `metrics` dict right?

Reply:
actually, added! @haritamar